package exception_handling;

public class ThrowsDemo {

	public static void divide(int a,int b) throws ArithmeticException {
		// TODO Auto-generated method stub
		int c = a/b;
		System.out.println("c values is"+ c);

	}

}
